web/html/src/manager/admin/setup/products/products.tsx

```diff
@@ -248,6 +250,36 @@ class ProductsPageWrapper extends React.Component {
       .catch(currentObject.handleResponseError);
   };
 
+  addOptionalChannels = (product, channels) => {
+    this.setState({ addingProducts: true, errors: [] });
+    Network.post("/rhn/manager/admin/setup/channels/optional", channels)
+      .then((data) => {
+        // returned data format is { channel : "error" }. If the value is null or missing the operation succeeded
+        let failedChannels = channels.filter((c) => data[c] != null);
+        let resultMessages: MessageType[] | null = null;
+        if (failedChannels.length === 0) {
+          resultMessages = MessagesUtils.success("Selected channels were scheduled successfully for syncing.");
```

Comment:
```suggestion
          resultMessages = MessagesUtils.success(t("Selected channels were scheduled successfully for syncing."));
```

Reply:
Damn I always forget one of them, even though I check every time :man_facepalming: